governance.md

```diff
@@ -18,88 +36,191 @@ The committee MUST:
+
+### Nominations Process
+
+The election officers may decide how to implement the nomination process, as long as the following requirements are fulfilled:
+
+- Nominations are publicly readable
+- Nominations are persisted over time and can be linked to for future reference
+- Nominations shall contain:
+  - The nominee’s email address, github handle, company affiliation, and CDEvents project(s) they contribute to
+  - For each of two contributors nominating this individual:
+    - The company they work for
+    - Their github handles
+    - The CDEvents project(s) they contribute to
+- Nominee must be nominated by contributors from at least two companies, and they can self-nominate
+- Once all requirements are satisfied, one election officer shall mark the nomination as accepted
+- Nominee may decline a nomination, though it is recommended to reach out to the nominee before posting a public nomination
+
+Two common ways of implementing nominations are either via a dedicated mailing list (e.g. `cdevents-governance@googlegroups.com` or through a dedicated GitHub issue in the community repo). 
+
+Any nominee who accepts the nomination will be on the ballot.
+
+### Election Process
+
+Elections will be held using time-limited [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on [CIVS](https://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method). The top vote getters will be elected to the open seats. This is the same process used by the Kubernetes project. Voters must opt into the CIVS system at https://civs1.civs.us/cgi-bin/opt_in.pl or they won't receive an invitation to vote.
+
+It may not always be possible for election officers to have access to all voters e-mail addresses. Nomination officials should setup a form or alternate mechanism for eligible voters to share their e-mail address for voting purposes.
+
+Details about the schedule and logistics of the election will be announced in a timely manner by the election officers to eligible candidates and voters via the cdevents-dev@googlegroups.com mailing list, on the #cdevents slack channel and at CDEvents working group meetings.
+
+Example timeline:
+
+1. Before or during nominations, send an email to all eligible voters to notify them that they are eligible (allowing people time to reach out if they believe they are eligible but are not on our list)
+1. Prompt eligible voters to opt into CIVS at https://civs1.civs.us/cgi-bin/opt_in.pl so they will be able to vote
+1. 1 week for nominations (starting on a Thursday until midnight UTC the next Wednesday)
+1. 1 week for the election itself (starting the following Thursday until midnight UTC the next Wednesday)
+
+### Election Officers
+
+For every election, the Governing Board wll choose two election officers, by the following criteria, so as to promote healthy rotation and diversity:
```

Comment:
Since the list contains some "should" and "should not", it could be the case that GB is not able to follow those criteria, but that would still be valid, or?
```suggestion
For every election, the Governing Board will choose two election officers, by the following criteria, so as to promote healthy rotation and diversity:
```

Reply:
fixed